File: components_page/components/dropdown/size.py
import dash_bootstrap_components as dbc
from dash import html

items = [
    dbc.DropdownMenuItem("First"),
    dbc.DropdownMenuItem(divider=True),
    dbc.DropdownMenuItem("Second"),
]

dropdown = html.Div(
    [
        dbc.DropdownMenu(
            label="large dropdown",
            size="lg",
            children=items,
            className="mb-3",
        ),
        dbc.DropdownMenu(label="normal dropdown", children=items, className="mb-3"),
        dbc.DropdownMenu(label="small dropdown", size="sm", children=items),
    ]
)

File: components_page/components/dropdown/size.R
library(dashBootstrapComponents)
library(dashHtmlComponents)

items_size <- list(
  dbcDropdownMenuItem("First"),
  dbcDropdownMenuItem(divider = TRUE),
  dbcDropdownMenuItem("Second")
)

dropdown <- htmlDiv(
  list(
    dbcDropdownMenu(
        label = "large dropdown",
        size = "lg",
        children = items_size,
        class_name = "mb-3"
    ),
    dbcDropdownMenu(
        label = "normal dropdown", children = items_size, class_name = "mb-3"
    ),
    dbcDropdownMenu(label = "small dropdown", size = "sm",
                    children = items_size)
  )
)
